c/CMakeLists.txt

```
@@ -1,74 +1,141 @@
 cmake_minimum_required(VERSION 3.9)
 
-project(libblake3 VERSION 0.1.0 DESCRIPTION "BLAKE3 C implementation")
-enable_language(C ASM)
+project(libblake3
+  VERSION 0.1.0
```

```suggestion
  VERSION 1.3.3
```
I guess it would make sense to set this to `1.3.3` to match `BLAKE3_VERSION_STRING`, right?